apps/vanilla/src/scripts/renderSortingAndOrdering.ts

```diff
@@ -0,0 +1,69 @@
+import { OrderOption } from '@js-camp/core/enum';
+import { PaginationOptions } from '@js-camp/core/models/paginationOptions';
+import { Sorting } from '@js-camp/core/models/sorting';
+import { setDefaultSelected } from '@js-camp/core/utils';
+
+import { DEFAULT_LIMIT, SORT_OPTIONS } from '../constants';
+import { KEY_ORDER, KEY_SORTING } from '../constants/key';
+import { LocalStorageService } from '../service/localStorage';
+
+import { renderListAnimeWithActivePage } from './renderPagination';
+
+const selectSort = document.querySelector<HTMLSelectElement>('.filter__item_select-sort');
+const selectOrdering = document.querySelector<HTMLSelectElement>('.filter__item_select-order');
+
+/** Init and render sorting and ordering list, hence render corresponding list anime.
+ * @param options Options of pagination.
+ */
```

Comment:
Please format **all** multiline jsdoc properly
```suggestion
/**
 * Init and render sorting and ordering list, hence render corresponding list anime.
 * @param options Options of pagination.
 */
```